tensorrt_llm/_torch/auto_deploy/custom_ops/attention/deepseek_v4_sparse_attention.py

```
@@ -0,0 +1,2132 @@
+        return _fake_fp4_act_quant(_hadamard_rotate(compressed), block_size=32)
+    nope, pe = torch.split(compressed, [nope_dim, rope_dim], dim=-1)
+    nope = _fake_fp8_act_quant(nope, block_size=64)
+    return torch.cat((nope, pe), dim=-1)
+
+
+def _overlap_transform_projected(
+    tensor: torch.Tensor,
+    head_dim: int,
+    value: float,
+) -> torch.Tensor:
+    batch_size, compressed_len, ratio, _ = tensor.shape
```

Ruff (0.15.15)

[warning] Unpacked variable `compressed_len` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)